### DISCUSSION TITLE
Test fails after installation (Version 4.2.1-27)

### DISCUSSION THREAD

**Discussion by smallrain1996** [neperfepx/neper#358] [Q&A] [answered]
Hello, I installed Neper version 4.2.0 on Ubuntu 20.04 and ran the `make test` and everything worked fine. However, after successfully installing the latest version (4.2.1-27) and running the `make test`, the following message appears.

```
99% tests passed, 2 tests failed out of 285

Total Test time (real) = 175.10 sec

The following tests FAILED:
	271 - V_dataelt3d (Child aborted)
	282 - V_imagebackground (Child aborted)
Errors while running CTest
make: *** [Makefile:107: test] Error 8
```
I can't understand what the errors 271 and 282 mean?
This problem is similar to #194 but I can't solve it by that method.

Thank you very much.

**Comment by xshang93** [comment, marked as answer]
I had the same issue. But I think there's nothing to be worried about. If you check the test log files, the reason these two fails is because `-scenebackground` and `-sceneshadow` are used in the tests. My guess is these two are either depreciated arguments that has been removed, or new arguments that has not yet been added, either way your neper would run without a problem.

**Reply by smallrain1996** [reply]
Thank you very much for your reply, this is very hard for an OCD person like me, but luckily Neper use is not affected.
